peyvandak/Majlesi/django-docker-compose-url-shortener/urlshortener_api/dashboard/models.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+from ipaddress import ip_address
+from pyexpat import model
+from statistics import mode
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Order(models.Model):
+    product_category = models.CharField(max_length=20)
+    payment_method = models.CharField(max_length=50)
```

Comment:
Just asking out of curiosity, what are these?? :D
payment method? shipping cost?

Reply:
I have forgotten to delete this model.